### DISCUSSION TITLE
ROCm on Radeon RX-580

### DISCUSSION THREAD

**Discussion by xclaim494** [ROCm/ROCm#3831] [Q&A]
I have a Radeon RX-580 Video Card on Windows 11 OS 64 Bit with Ubuntu 22.04.5 LTS - Kernel is 5.15.153 running in WSL2 container - I apologize for any stupid questions - But can i get ROCm to work on my machine ? I've heard that the latest ROCm might not work but maybe i need to install an older version? I want to start dabbling with AI, ML, LLM etc etc and cant justify buying a new VC just yet.

Please can you share exact steps to get it working so that i can use my GPU ? TY

**Comment by fluidnumericsJoe** [comment]
I wouldn't advise using windows at all, to start. Try Ubuntu installed directly on your system; it's not hard to use or get started with at all. Second, check out the ROCm docs ( rocm.docs.amd.com ). All of the installation guides are there. You may specifically find the [supported GPU list](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html) valuable.

The Radeon RX-580 is a GCN 4 (gfx800 series) GPU and is a bit out of date; the oldest supported series is gfx900 . Keep in mind that "supported" simply means that AMD tests ROCm builds on the "supported" GPUs; it doesn't mean that yours won't work. We run a few unsupported GPUs in the gfx900 series at our shop with no problem. You can certainly try to get things going following documentation as closely as possible for your setup, but your mileage may vary, particularly with the older model GPUs. You may have reasonable luck with an older version of ROCm (e.g.. ROCm 5.x, rather than ROCm 6.x).

Yes, I know, shouldn't AMD support all of it's GPUs ?
